docs/going-into-production.rst

```diff
@@ -0,0 +1,357 @@
+.. _going-into-production:
+
+=====================
+Going into production
+=====================
+
+Running CrateDB in different environments requires different approaches. This
+document outlines the basics you need to consider when going into production.
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+.. _prod-bootstrapping:
+
+Configure bootstrapping
+=======================
+
+The process of forming a cluster is known as *bootstrapping*. Consult the
+how-to guide on :ref:`CrateDB multi-node setups <multi_node_setup>` for an
+overview of the two different ways to bootstrap a cluster.
+
+If you have been using CrateDB for development on your local machine, there is
+a good chance you have been using :ref:`single host auto-bootstrapping
+<auto-bootstrapping>`.
+
+For improved performance and resiliency, you should run production CrateDB
+clusters with one node per host machine. To do this, you must manually
+configure the bootstrapping process by telling nodes how to:
+
+  a. :ref:`Discover other nodes <discovery>`, and
+  b. :ref:`Elect a master node <master-node-election>`
+
+This process is known as *manual bootstrapping*. See the :ref:`how-to guide
+<manual-bootstrapping>` for more information about how to bootstrap a cluster
+manually.
+
+Switching to a manual bootstrapping configuration is the first step towards
+going into production.
+
+
+.. _prod-naming:
+
+Naming
+======
+
+
+.. _prod-cluster-name:
+
+Configure a logical cluster name
+--------------------------------
+
+The `cluster.name`_ setting allows you to override the default cluster name of
+``crate``. You should use this setting to give a logical name to your cluster.
```

Comment:
This is maybe something for the configuration section, but do users need to follow a pattern?

Reply:
I've added a note suggesting a pattern directly under this para